WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -82,6 +91,21 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         NotificationCenter.default.addObserver(self, selector: #selector(showAddSelfHostedSite), name: .addSelfHosted, object: nil)
     }
 
+    private func setupSegmentedControl() {
+        segmentedControlContainerView.isHidden = !FeatureFlag.mySiteDashboard.enabled
```

Comment:
What do you think about this flag being always `false`?

Given this is in the initial stages and only you and I will be working on it, I think it might confuse people to see this intermediate stage for quite so long there.

Reply:
Fixed! d2da643